src/ecs/input/bundle.rs

```diff
@@ -0,0 +1,48 @@
+//! ECS input bundle
+
+use app::ApplicationBuilder;
+use config::Config;
+use ecs::ECSBundle;
+use ecs::input::{Bindings, InputHandler};
+use error::Result;
+
+/// Bundle for adding the InputHandler and input bindings
+///
+/// # Errors
+///
+/// No errors returned from this bundle.
+///
+pub struct InputBundle {
+    bindings: Option<Bindings>,
+}
+
+impl InputBundle {
+    /// Create a new input bundle with no bindings
+    pub fn new() -> Self {
+        Self { bindings: None }
+    }
+
+    /// Use the provided bindings with the InputHandler
+    pub fn with_bindings(mut self, bindings: Bindings) -> Self {
```

Comment:
For the sake of a flatter syntax, could we have `new(Bindings)` and `from_file` instead? I wonder if we'd want no bindings at all.

Reply:
From what I've understood, the InputHandler only use bindings for actions/axes, but the user can also access the internal storages directly for a lot of things, so you aren't required to use bindings, which is why I made that a separate function.